src/components/sandbox/ComponentSerializer.ts

```diff
@@ -85,6 +86,10 @@ export class ComponentSerializer {
       .filter(item => !!item)
       .join(" ");
 
+      if (ReactiveComponents.includes(elementType) && (this.serializer instanceof AngularReactiveSerializer)) {
```

Comment:
I think you forgot to move this to the AngularReactiveSerializer postProcess function

Reply:
As discussed, although can't move this to postProcess, however I moved this out to modifyProps method, and now the implementation within the AngularReactiveSerialzer is going to take care of this, no longer need these imports or this logic.